app/src/main/java/org/mozilla/tv/firefox/hint/InactiveHintViewModel.kt

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.hint
+
+import io.reactivex.Observable
+
+/**
+ * Used when a user is part of an experiment that does not show the hint bar
+ */
+class InactiveHintViewModel: HintViewModel {
```

Comment:
Do we need a separate `ViewModel` for this? Shouldn't `OverlayHintVM` handle this?

Reply:
I originally handled it inside the other VMs, but it wound up looking pretty messy.  Also this is the same for `OverlayHintVM` and `WebRenderHintVM`.  This also keeps our A/B testing process more consistent with our existing A/A (the fragment instantiates a class based on the test, the rest of the app is unaware of Fretboard).